engine/table/src/main/java/io/deephaven/engine/table/impl/select/WhereFilterFactory.java

```diff
@@ -327,8 +333,11 @@ private static WhereFilter getSelectFilter(String colName, String quickFilter, Q
             return ComparableRangeFilter.makeBigDecimalRange(colName, quickFilter);
         } else if (filterMode != QuickFilterMode.NUMERIC) {
             if (colClass == String.class) {
-                return new StringContainsFilter(MatchFilter.CaseSensitivity.IgnoreCase, MatchFilter.MatchType.Regular,
-                        colName, quickFilter);
+                return new PatternFindFilter(
+                        ColumnName.of(colName),
+                        Pattern.compile(Pattern.quote(quickFilter), Pattern.CASE_INSENSITIVE),
```

Comment:
This matches the behavior before (essentially, simplifying the logic in the values.length == 1 case); I wonder, should we instead be doing:

```java
Pattern.compile(quickFilter, Pattern.CASE_INSENSITIVE | Pattern.LITERAL)
```
?

Reply:
Choosing to keep the `.quote` for now, since I know it's equivalent to the previous logic.